xilem/examples/emoji_picker.rs

```diff
@@ -91,19 +91,25 @@ fn paginate(
     count_per_page: usize,
     max_count: usize,
 ) -> impl WidgetView<usize> {
-    let percentage = (current_start * 100) / max_count;
+    let current_end = (current_start + count_per_page).min(max_count);
+    let percentage_start = (current_start * 100) / max_count;
+    let percentage_end = (current_end * 100) / max_count;
 
     flex((
         // TODO: Expose that this is a previous page button to accessibility
-        button("<-", move |data| {
-            *data = current_start.saturating_sub(count_per_page);
+        (current_start != 0).then(|| {
```

Comment:
Ideally, we'd disable the button instead of hiding it, but that's not currently exposed to Xilem (it will likely end up looking similar to transform)